packages/eslint-plugin/src/rules/no-lifecycle-call.ts

```diff
@@ -0,0 +1,46 @@
+import { createESLintRule } from '../utils/create-eslint-rule';
+import { isAngularLifecycleMethod, isIdentifier } from '../utils/utils';
+
+type Options = [];
+export type MessageIds = 'noLifecycleCall';
+export const RULE_NAME = 'no-lifecycle-call';
+
+export default createESLintRule<Options, MessageIds>({
+  create(context) {
```

Comment:
Sorry that it's a fairly trivial thing, but in all other cases the `create()` method comes last, so that all the metadata is at the top, please could we follow that here as well?